mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/MainActivity.java

```diff
@@ -103,9 +107,18 @@ public void onBackPressed() {
                 .findFragmentById(R.id.bottom_navigation_fragment_container);
         if (fragment != null && !(fragment instanceof HomeFragment) && fragment.isVisible()) {
             navigateFragment(R.id.action_home, true);
-            return;
+        } else if (!doubleBackToExitPressedOnce) {
+            this.doubleBackToExitPressedOnce = true;
+            Toast.makeText(this, R.string.exit_message, Toast.LENGTH_SHORT).show();
```

Comment:
Use our in-house library `Toaster` instead.

Reply:
@luckyman20 done